modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/Tooltip.kt

```diff
@@ -69,10 +63,7 @@ fun TooltipPopup(
         properties = properties,
         onDismissRequest = onClickOutside,
     ) {
-        AnimatedVisibility(
-            visible = show,
-            enter = fadeIn(),
-            exit = fadeOut(),
+        Box(
```

Comment:
The changes to the `TooltipPopup` were made for performance reasons. The caller is now responsible for including it when needed. Having `PopupWindow` always present caused significant lag when multiple popups were added, even if they weren’t displayed, because multiple windows were being created.